stdlib/private/SwiftPrivateLibcExtras/SwiftPrivateLibcExtras.swift

```diff
@@ -37,36 +37,36 @@ public var _stdlib_FD_SETSIZE: CInt {
   return 1024
 }
 
-public struct _stdlib_fd_set {
-  var _data: [UInt32]
+public struct __stdlib_fd_set<T: UnsignedInteger> {
```

Comment:
Also, the standard library style is to put spaces on both sides of the colon in generic constraints (`<T : UnsignedInteger>`).